packages/ocs/dashboards/persistent-internal/details-card.tsx

```diff
@@ -34,6 +40,12 @@ const DetailsCard: React.FC = () => {
     StorageClusterModel,
     odfNamespace
   );
+
+  const [cephData, cephLoaded, cephLoadError] = useSafeK8sList<CephClusterKind>(
+    CephClusterModel,
+    getNamespace(ocsData?.[0])
```

Comment:
```suggestion
    odfNamespace
```
Rebase this PR on top of the master and use the namespace variable odfNamespace